ui/src/components/SampleView/PhaseInput.jsx

```diff
@@ -20,14 +20,21 @@ function PhaseInput() {
       value={value}
       data-busy={isBusy || undefined}
       onChange={(evt) => {
-        if (evt.target.value !== 'Unknown') {
-          dispatch(changeCurrentPhase(evt.target.value));
-        }
+        dispatch(changeCurrentPhase(evt.target.value));
       }}
     >
-      {options.map((option) => (
-        <option key={option}>{option}</option>
-      ))}
+      {/*
+        Makes it so that "Unknown" may appear to describe
+        the current phase, but may not be selected.
+       */}
+      <option hidden key="Unknown">
+        Unknown
+      </option>
+      {options
+        .filter((option) => option !== 'Unknown')
+        .map((option) => (
+          <option key={option}>{option}</option>
+        ))}
```

Comment:
Makes total sense, but I think this would work too:

```suggestion
      {options.map((option) => (
        <option key={option} hidden={option === 'Unknown')>{option}</option>
      ))}
```

Reply:
If you want to remove unknown, you could do it in a specific diffractometer HO.